lib/engine/step/special_token.rb

```diff
@@ -9,6 +9,7 @@ class SpecialToken < Base
       include Tokener
 
       def actions(entity)
+        return [] unless ability(entity)
```

Comment:
why did you add this, it's the same as line 13

Reply:
Right - removed.